pkg/agent/secondarynetwork/podwatch/controller.go

```diff
@@ -114,6 +121,49 @@ func podKeyGet(pod *corev1.Pod) string {
 	return pod.Namespace + "/" + pod.Name
 }
 
+// buildVFDeviceIDListPerPod is a helper function to build a cache structure with the
+// list of all the PCI addresses allocated per Pod based on their resource requests (in Pod spec).
+// When there is a request for a VF resource (to associate it for a secondary network interface),
+// getUnusedSriovVFDeviceIDPerPod will use this cache information to pick up a unique PCI address
+// which is still not associated with a linux interface name.
```

Comment:
Nit: linux -> Linux. Could just say device name, in case later we reuse the func for Windows.